renderdoc/api/replay/renderdoc_replay.h

```diff
@@ -170,7 +170,7 @@ struct IReplayOutput
   virtual ResourceId GetCustomShaderTexID() = 0;
   virtual bool PickPixel(ResourceId texID, bool customShader, uint32_t x, uint32_t y,
                          uint32_t sliceFace, uint32_t mip, uint32_t sample, PixelValue *val) = 0;
-  virtual uint32_t PickVertex(uint32_t eventID, uint32_t x, uint32_t y) = 0;
+  virtual uint32_t PickVertex(uint32_t eventID, uint32_t x, uint32_t y, uint32_t *pickedInstance) = 0;
```

Comment:
I'm not suggesting it has to be part of this PR but I wonder if we would want this to change to be a `PickedVertex` struct return that contains vertex ID, instance ID, and potentially other data like primitive ID.